sentry_sdk/tracing_utils.py

```diff
@@ -109,7 +109,7 @@ def has_tracing_enabled(options):
     # type: (Dict[str, Any]) -> bool
     """
     Returns True if either traces_sample_rate or traces_sampler is
-    non-zero/defined, False otherwise.
+    defined, False otherwise.
```

Comment:
this was changed originally here and was lost in later refactors so I brought it back
https://github.com/getsentry/sentry-python/pull/1065

Reply:
Good catch.